_posts/2019-08-13-release-1.9.0.md

```diff
@@ -0,0 +1,362 @@
+Table API or SQL queries. More details are available in
+[FLINK-10556](https://issues.apache.org/jira/browse/FLINK-10556).
+
+While, previously, table definitions for Table API or SQL queries were always
+volatile, the new catalog connector additionally allows persisting a table in
+Metastore that is created with a SQL DDL statement (see above). This means
+that you connect to Metastore and register a table that is, for example,
+backed by a Kafka topic. From now on, you can query that table whenever your
+catalog is connected to Metastore.
+
+Please note that the Hive support in Flink 1.9 is experimental. We are
+planning to stabilize these features for the next release and are looking
+forward to your feedback.
+
+
+### Preview of the new Python Table API (FLIP-38)
+
+This release also introduces a first version of a Python Table API
+([FLIP-38](https://cwiki.apache.org/confluence/display/FLINK/FLIP-38%3A+Python+Table+API)).
+This marks the start towards our goal of bringing
+full-fledged Python support to Flink. The feature was designed as a slim
+Python API wrapper around the Table API, basically translating Python Table
+API method calls into Java Table API calls. In the initial version that ships
+with Flink 1.9, the Python Table API does not support UDFs yet, but just
+standard relational operations. Support for UDFs implemented in Python is on
+the roadmap for future releases.
+
+If you’d like to try the new Python API, you have to manually [install
+PyFlink](https://ci.apache.org/projects/flink/flink-docs-release-1.9/flinkDev/building.html#build-pyflink).
+From there, you can have a look at [this
+walkthrough](https://ci.apache.org/projects/flink/flink-docs-release-1.9/tutorials/python_table_api.html)
+or explore it on your own. The [community is currently
+working](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/VOTE-Publish-the-PyFlink-into-PyPI-td31201.html)
+on preparing a `pyflink` Python package that will be made available for
+installation via `pip`.
+
+
+## Important Changes
+
+ * The Table API and SQL are now part of the default configuration of the
+   Flink distribution. Before, the Table API and SQL had to be enabled by
+   moving the corresponding JAR file from ./opt to ./lib.
+ * The machine learning library (flink-ml) has been removed in preparation for
+   [FLIP-39](https://docs.google.com/document/d/1StObo1DLp8iiy0rbukx8kwAJb0BwDZrQrMWub3DzsEo/edit).
+ * The old DataSet and DataStream Python APIs have been removed in favor of
+   [FLIP-38](https://cwiki.apache.org/confluence/display/FLINK/FLIP-38%3A+Python+Table+API).
+ * Flink can be compiled and run on Java 9. Note that certain components
+   interacting with external systems (connectors, filesystems, reporters) may
+   not work since the respective projects may have skipped Java 9 support.
+
+
+## Release Notes
+
+Please review the [release
+notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12344601)
```

Comment:
Should we use this link?
https://ci.apache.org/projects/flink/flink-docs-release-1.9/release-notes/flink-1.9.html

Reply:
Yes, I will update once Gordon's PR is merged: https://github.com/apache/flink-web/pull/244#discussion_r313414497

Reply:
I will address this when merging.